src/helpers/pagination.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @class
+ * @classdesc - Pagination Class
+ */
+class Pagination {
+  /**
+   * @constructor
+   * @memberof Pagination
+   * @param {number|string} page
+   * @param {number|string} limit
+   */
+  constructor(page = 1, limit = 10) {
```

Comment:
What happens if a user sets a value of `0` or negative value for `page`?

Reply:
What happens if page is set to `five`?

Reply:
same for `limit`

Reply:
<img width="1440" alt="Screenshot 2019-07-09 at 5 16 23 AM" src="https://user-images.githubusercontent.com/47529849/60859084-01327480-a209-11e9-97fd-49fd1522f593.png">
<img width="1440" alt="Screenshot 2019-07-09 at 5 19 26 AM" src="https://user-images.githubusercontent.com/47529849/60859131-38088a80-a209-11e9-8842-0ede96d3958b.png">

<img width="1440" alt="Screenshot 2019-07-09 at 5 39 28 AM" src="https://user-images.githubusercontent.com/47529849/60860391-28d80b80-a20e-11e9-9d21-df903e578312.png">


The page = 1 and limit = 10 in the constructor is just default setup. So whenever the database is queried, the limit is gotten from the querymeta. Though It can take in any number assigned to it but returns an appropriate error message when a negative number is assigned to it. However, if you don't set the limit, it will be 10 by default.

Same with the page.

Reply:
So, on a second thought, I think I can actually validate the query such that if a negative value or a character is assigned to either the page or the limit, it should return the default setting of page =1 and limit=10, instead of returning an error message or eventually crashing the app.

Reply:
<img width="1440" alt="Screenshot 2019-07-09 at 7 11 43 AM" src="https://user-images.githubusercontent.com/47529849/60863520-e23bde80-a218-11e9-972b-b7f8639eb9b9.png">